tools/cli/main.c

```diff
@@ -455,6 +457,17 @@ TDNFCliInvokeRepoSync(
     return TDNFRepoSync(pContext->hTdnf, pRepoSyncArgs);
 }
 
+uint32_t
+TDNFCliInvokeRepoQuery(
+    PTDNF_CLI_CONTEXT pContext,
+    PTDNF_REPOQUERY_ARGS pRepoQueryArgs,
+    PTDNF_PKG_INFO *ppPkgInfos,
+    uint32_t *pdwCount
+    )
+{
```

Comment:
That's redundant. If you look at the other functions in that file you will see that they are also just one liners passing args without change to another function. Arg checking should take place there.

Normally I would agree because it's a common pattern in tdnf to check arguments over and over again. But here in this file, the pattern is not to, if we just pass the args on.